e2e/utils.sh

```diff
@@ -22,6 +22,19 @@ install_shared_asset() {
     cp -R "$ASSET_ROOT"/* "$(dirname "$E2E_NETWORKS_JSON")"
 }
 
+dfx_extension_install_manually() (
+    cd "$GIT_ROOT_DIR"
+    local extension_name="$1"
+    package_version=$(HOME="$CARGO_HOME" cargo metadata --format-version=1 | jq -r '.workspace_members[]' | rg "$extension_name" | cut -d" " -f2)
```

Comment:
Is there some special functionality uses here that rg provides that grep doesn't?
```suggestion
    package_version=$(HOME="$CARGO_HOME" cargo metadata --format-version=1 | jq -r '.workspace_members[]' | grep "$extension_name" | cut -d" " -f2)
```

Reply:
no